tools/devops/automation/build-pipeline.yml

```diff
@@ -234,7 +234,8 @@ stages:
 
 - stage: build_packages
   displayName: 'Build'
-  dependsOn: []
+  dependsOn:
+  - build_packages
```

Comment:
You can use `dependsOn` with cyclic reference to the same stage?!

Reply:
nice catch 🤦